backend/django_core/static_src/components/page/upload/Login.svelte

```diff
@@ -2,6 +2,7 @@
     import { createEventDispatcher } from "svelte";
     import Chevron from "$icons/Chevron/Index.svelte";
     import Info from "$icons/Info/Index.svelte";
+    import FishGradient from "../../../public/images/fish/fish-gradient.png";
```

Comment:
This should be dynamically genreated

Reply:
> This should be dynamically genreated

Why? there is no need, we need to import fish image, then why dont we just import fish with already gradient image? why extra work